app/oess_pull_aws_interfaces.pl

```diff
@@ -177,6 +177,7 @@ sub get_vrf_aws_details {
         warn "Found\n";
         return $aws
     }
+    return;
```

Comment:
I don't understand what actually is changing here

Reply:
Without `return;` the sub returns `''` which evaluates as `defined`.

Reply:
Without it `update_endpoint_values` is called even when aws details can't be found.

Reply:
oh I gotcha now. Thanks